recipe/build.sh

```diff
@@ -3,6 +3,9 @@
 if [ "$(uname)" == "Darwin" ]; then
     export CXX="${CXX} -stdlib=libc++"
     export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
+else
+    # Shared Fortran libs on OSX unsupported as of 1.8.18
+    FORTRAN_FLAGS=("--enable-fortran" "--enable-fortran2003")
```

Comment:
Nice! Thanks for figuring that out. I think removing `libgfortran` is unnecessary then as we know it will fail if it does that.